assets/html/themeswap.js

```diff
@@ -44,23 +44,23 @@ function set_theme_from_local_storage() {
     // Store the style sheets of inactive themes so that we could disable them
     if(themename !== theme) disabled.push(ss);
   }
-  if(active !== null) {
-    // If we did find an active theme, we'll (1) add the theme--$(theme) class to <html>
-    document.getElementsByTagName('html')[0].className = "theme--" + theme;
-    // and (2) disable all the other theme stylesheets
-    disabled.forEach(function(ss){
-      ss.disabled = true;
-    });
-  }
-  else if(darkTheme !== null && darkPreference === true) {
-    // If we did find an active theme, we'll (1) add the theme--$(theme) class to <html>
-    document.getElementsByTagName('html')[0].className = "theme--" + darkTheme;
-    // and (2) disable all the other theme stylesheets
-    disabled.forEach(function(ss){
+  if (theme === "auto" || (active === null && darkPreference === true && darkTheme !== null)) {
```

Comment:
Umm, `theme` comes from local storage, and will never be `"auto"`, because we never set it to that.

Pretty sure this script does not need to be changed. I think it should be enough to add an "auto" dropdown item to the theme switcher, that wipes the value from `localStorage` when you switch to it. I.e. in here: https://github.com/JuliaDocs/Documenter.jl/blob/master/assets/html/js/themes.js

Reply:
Ohhk i added the "auto" dropdown item there, please verify if its ok!!|
https://github.com/JuliaDocs/Documenter.jl/pull/2085#issue-1637542656